### ISSUE TITLE
SQL Editor Cursor

### ISSUE THREAD

**Issue by metinuzuner** [madorin/fibplus#80]
Hello,
Thanks for developing FIBPlus.
I'm using C++ Builder XE6. FIBPlus Component is installed without any problems.
When the pFIBDataSet1->SQl Generator screen is first opened (SelectSQL and ModifySQL), there is a Cursor, but when something is written to TMemo (I think fraSQLEdit.pas) where SQL is written, the Cursor disappears. Also, Scroll bars flicker while typing SQL.

Thanks.
Note: I am writing using Google translate.

![fibplus-Sql-Editor](https://github.com/madorin/fibplus/assets/83384319/a789ade8-3090-4421-a286-23a534980159)

**Comment by madorin** [comment]
@metinuzuner , please check if this branch /madorin/fibplus/tree/80-editor-cursor works in your C++ Builder XE6 and let me know

**Comment by metinuzuner** [comment]
Thank you for your answer. The problem is found in section 6845 of the pFIBSyntaxMemo.pas parts;

FDefined: TStringsDefined;
FDelimiter: Character;
FQuoteChar: Character;
UpdateCount: Integer;

code

// FDefined: TStringsDefined;
// FDelimiter: Character;
// FQuoteChar: Character;
   UpdateCount: Integer;
   
When I changed it to , the cursor problem went away.

The retarder was already showing the following warning. The defined variables have never been used.
When I canceled the variable definitions, the component worked smoothly.

[dcc32 Hint] pFIBSyntaxMemo.pas(6846): H2219 Special symbol 'FDefined' declared but never used
[dcc32 Hint] pFIBSyntaxMemo.pas(6847): H2219 Special symbol 'FDelimiter' declared but never used
[dcc32 Hint] pFIBSyntaxMemo.pas(6848): H2219 Special symbol 'FQuoteChar' declared but never used

Thank you again. Enjoy your work.
